source/support.tex

```diff
@@ -3317,6 +3325,7 @@
 
 \indexlibraryglobal{source_location}%
 \begin{codeblock}
+// all freestanding
```

Comment:
This is the class synopsis. The header synopsis is above.

Reply:
fixed